src/kirby/components/avatar/avatar.component.tns.ts

```diff
@@ -30,40 +30,38 @@ export class AvatarComponent extends ContentView {
 
   view: View;
 
-  constructor(private zone: NgZone) {
+  constructor() {
     super();
   }
 
   onViewLoaded(args: EventData) {
     this.view = <View>args.object; // We need a reference to the view so we can access it on orientation changes
-    // this.setupOnOrientationChangeListener();
     this.addShadow();
+    this.setClipping();
   }
 
-  setupOnOrientationChangeListener() {
-    app.on(app.orientationChangedEvent, (args: OrientationChangedEventData) => {
-      // Run in the zone, to make sure Angular data binding is informed of this:
-      this.zone.run(() => this.addShadow());
-    });
-  }
-
-  addShadow() {
+  private addShadow() {
     if (this.shadow) {
       setTimeout(() => {
         if (this.view.android) {
-          this.view.eachChildView((child) => {
-            const bgColor = child.style.backgroundColor;
-            const transparentColor = '#00ff0000';
+          this.view.eachChildView((child: View) => {
             const androidView = child.android;
-            const shape = new android.graphics.drawable.GradientDrawable();
-            shape.setShape(android.graphics.drawable.GradientDrawable.RECTANGLE);
-            shape.setColor(
-              android.graphics.Color.parseColor(bgColor ? bgColor + '' : transparentColor)
-            );
-            shape.setCornerRadius(androidView.getMeasuredWidth());
-            androidView.setBackgroundDrawable(shape);
-            androidView.setClipToOutline(true);
-            androidView.setElevation(ScssHelper.ELEVATION_IMAGE);
+            const cssClasses = child.cssClasses;
```

Comment:
I guess this could be fixed by the new `[kirbyElevation] directive instead. See Card for an example.